src/frontend/hooks/goose-form.js

```diff
@@ -81,13 +81,25 @@ const useGooseForm = () => {
 
   const saveForm = useCallback(async () => {
     // TODO: Validation checks here.
-
     // Send data to server:
+
     try {
+      // files added 
       const files = images;
-      let imageUrls = [];
-
-      if (files.length > 0) {
+  
+      // all files uploaded 
+      let imageUrls2 = [];
+      
+      if (files.length > 0 || imageUrls.length > 0) {
```

Comment:
Do we need to enter this block if `imagesUrls.length > 0`? Because we only want to upload new images, which is represented by `files`. `imageUrls` is for images that are already uploaded